### DISCUSSION TITLE
Visual Feedback, when the time scroll bar is dragged

### DISCUSSION THREAD

**Discussion by smp-marco** [smplayer-dev/smplayer#571] [Ideas]
**Visual Feedback, when the time scroll bar is dragged**

Having loaded a video for analysis, into SMP, I then realise/remember that I need to also load the video in YouTube.
This in order to find the point of interest.
By that I mean, that I will drag the timer scroll bar, watching the thumbnails of the events unfold.
When I find the clip start point, I'll note the time, and then, in SMP, drag the time scroll bar to that point.

It would be great, if either the video moved with the scroll bar, or, as in YT, we could see thumbnails of perhaps the I frames, in order to locate the region of interest, in the video.

Evidently, if analysing a YT video, the searching can be done in YT, with just a few extra steps.
However; with a video off the hard drive; finding the point of interest will not be easy.
:)

**Comment by smplayer-dev** [comment]
Preferences -> Interface -> Seeking -> Behaviour of time slider: seek to position while dragging.

**Reply by smp-marco** [reply]
That's great Ricardo :)
Just tested it now.

I looked through preferences before making the post.
I must have been blind :(
It was late ... Hahah I should have been in bed :D
